### DISCUSSION TITLE
Selection Prompt and Live table

### DISCUSSION THREAD

**Discussion by shuzERR** [spectreconsole/spectre.console#1035] [General]
Hello, I wanna create application where I could show LiveTable on the top and still use SelectionPrompt or any other features on the bottom of the same window. Is that possible?

Something like that: 

![obraz_2022-10-31_145242412](https://user-images.githubusercontent.com/36573380/199024104-14ff7d6b-e98d-4ad9-ae38-a956cd2e4f57.png)

**Comment by patriksvensson** [comment]
@shuzERR No that is not possible.

**Reply by shuzERR** [reply]
Maybe some other possibility to see two features at the same time? Something like one window with two other working inside?

**Reply by patriksvensson** [reply]
Not any way that I can think of since prompts, status, progress and other live widgets can't run more than one at the time. This is by design.

I think your only option here is to implement your own widget from scratch that does what you want it to do.
